docs/external/known-issues.md

```diff
@@ -10,6 +10,7 @@ This section lists the major known issues with vSphere Docker Volume Service. Fo
 -  Plugin fails to create volumes after installation on VM running boot2docker Linux. This is because open-vm-tools available for boot2docker doesn't install the vSockets driver and hence the plugin is unable to contact the ESX service. [#1744](https://github.com/vmware/docker-volume-vsphere/issues/1744)
 -  Currently "vmdk-opsd stop" just stops (exits) the service forcefully. If there are operations in flight it could kill them in the middle of execution. This can potentially create inconsistencies in VM attachement, KV files or auth-db. [#1073](https://github.com/vmware/docker-volume-vsphere/issues/1073)
 -  vDVS volume that has already been created cannot be resized or it's size cannot be updated.
+-  For Windows plugin, if the volume is mounted using it's short name then subsequent mount requests for the same volume should only use short volume name. Same thing applies for long volume names. Otherwise user will run into the issue [#2006](https://github.com/vmware/docker-volume-vsphere/issues/2006)
```

Comment:
let's wait for https://github.com/vmware/docker-volume-vsphere/pull/2011 to be merged, I have added the section for Windows plugin; you have to add this information into the section.